modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -20,6 +20,10 @@ The `oc adm upgrade status` command can be used for clusters from version 4.12 u
 Your cluster does not need to be a Technology Preview-enabled cluster in order for you to use the `oc adm upgrade status` command. 
 ====
 
+.Prerequisites
+
+* Your cluster is connected to an update service 
```

Comment:
The Update Service is about helping you figure out which release you want to update to next.  I don't think it matters here for `status`, because `status` is for when you've already launched the update and are trying to see how the already-in-progress update is going.

Reply:
+1 `status` cmd is for watching the progress during an upgrade, whether it's connected to an upstream or not .
cc @dis016